tests/test_operation_logs.py

```diff
@@ -34,4 +35,18 @@ def test_verified_reports(self):
         results = self._results(api.url_for(OperationsLogView, candidate_committee_id='02', status_num=1))
         self.assertEqual(len(results), 1)
         for result in results:
-            self.assertTrue(1, result['status_num'])
\ No newline at end of file
+            self.assertTrue(1, result['status_num'])
+
+    def test_receipt_date(self):
```

Comment:
Well done @hcaofec! Could you add another test case to test the data range on receipt_date column? A simple test to make sure that the date range is picked up correctly.Here is an example to test the data range filter:
https://github.com/fecgov/openFEC/blob/develop/tests/test_filings.py#L35

Reply:
test case has been added.
@fec-jli @pkfec

Reply:
@hcaofec Looks good. I will merge this PR.